src/graphql/queries/extensions.graphql

```diff
@@ -15,3 +15,39 @@ query GetExtensionInstallationsCount($id: ID!) {
     reputationWeighted
   }
 }
+
+query GetAllColoniesExtensions(
+  $contributorAddress: ID!
+  $sortDirection: ModelSortDirection = ASC
+  $limit: Int = 100
+  $nextToken: String
+  $isWatching: Boolean
```

Comment:
It feels like this argument doesn't belong here anymore, should the query always set the filter to true instead?